unitTestSupportFunctions/calculateDrag.m

```diff
@@ -0,0 +1,30 @@
+%% Function documentation
+%
+% Calculate drag force on a single body with no-slip boundary conditions in a rectangular domain
+%
+%               Input :
+%           FComplete : The complete force vector
+%              fldMsh : Nodes and elements for the fluid mesh
+%              homDBC : The global numbering of the DOFs where homogeneous
+%                       Dirichlet boundary conditions are applied
+%          parameters : Flow parameters
+%
+%
+%              Output :
+%                  Fx : Force (drag) on the body in x-direction
+%
+%% Function main body
+function Fx = calculateDrag (FComplete, fldMsh, homDBC, parameters)
```

Comment:
Add information about the problem dimensionality, that would be another input variable such as noSpatialDimensions. In this case it should be set to 3 (ux, uy, uz) but later it can be extended to 4 for 3D problems